src/handlers/componentInstall.ts

```diff
@@ -42,6 +43,17 @@ export default async function componentInstall(name: string): Promise<void> {
     );
   }
 
+  // Make sure the given system is installed and has the correct branch/commit/tag checked out.
+  try {
+    await cloneIntoCache('systems', [systemName])(emulsifyConfig.system);
```

Comment:
This will clone the system into the `~/.emulsify` folder if doesn't already exist.